app/controllers/buyers/applications_controller.rb

```diff
@@ -36,6 +35,10 @@ def create
 
   protected
 
+  def accessible_not_bought_cinstances
```

Comment:
This naming is quite confusing... :grimacing:  Especially looking at the implementation. I know that it was already like this. But dunno, maybe we could rename? Something like `accessible_buyers_cinstances` to reflect that it represents the apps that were bought by the buyer (developer), and not the provider itself (which is, as I understand, the meaning of this).

Reply:
Also, this added scope is new (and not moved from somewhere), and I was wondering - how did it use to filter? I guess by the `define_search_scope` method? So, I'm wondering if these two are redundant now.

And the same actually for the service controller.

Reply:
I couldn't figure out what "accessible_not_bought_cinstances" meant but if you think your suggestion makes more sense I'm all for changing it.

About the scope thingy, the apps under "account" scope buyer/accounts/applications or whatever is filtered by injecting the account id as a hidden parameter, which is very weird IMO. I noticed this `super.bought_by(@account)` in some other controller (or it was maybe in the model) and I thought it is the "right" way to do it.